build_tools/github_actions/runner/gcp/create_image.sh

```diff
@@ -149,24 +197,21 @@ function create_image() {
   echo "Streaming startup logs from instance to stdout and ${log_file}"
 
   # Get the PID of the startup script
-  local startup_pid="$(gcloud compute ssh "${INSTANCE_NAME}" --zone="${ZONE}" \
-      --no-user-output-enabled \
-      --command='systemctl show --property=ExecMainPID --value google-startup-scripts')"
+  local startup_pid="$(instance_ssh --command='systemctl show --property=ExecMainPID --value google-startup-scripts')"
 
   echo ""
   echo "*******************"
 
   # -t forces a pseudo-tty which allows us to run tail with a follow
   gcloud compute ssh "${INSTANCE_NAME}" --zone="${ZONE}" \
-      --no-user-output-enabled \
-      --ssh-flag="-t" \
-      --command="tail --follow=name --retry --pid=${startup_pid} /startup.log" \
-      | tee "${log_file}"
+      --no-user-output-enabled --ssh-flag="-t" \
+      --command="tail --follow=name --retry --lines=+1 --pid=${startup_pid} /startup.log"
+      # | tee "${log_file}"
```

Comment:
here's a suggested change uncommenting the logging line (change LGTM otherwise, but automerge...)

```suggestion
      --no-user-output-enabled --ssh-flag="-t" \
      --command="tail --follow=name --retry --lines=+1 --pid=${startup_pid} /startup.log"
      | tee "${log_file}"
```